docs/src/maintain/manage-releases.md

```diff
@@ -49,6 +49,31 @@ In rare cases, a second patch release might be necessary if the release is known
 
 After the patch release has been published (or no patch release is necessary), close the release issue and inform the team that they can start merging in semver-minor changes again.
 
+### Release Parameters
+
+The following table shows examples of the option to select as `RELEASE_TYPE` when starting `eslint-js Release` (the `@eslint/js` package release) and `eslint Release` (the `eslint` package release) jobs on Jenkins to release a new version with the latest features. In both jobs, `main` should be selected as `RELEASE_BRANCH`.
+
+| **Current version** | **Releasing** | **`eslint-js Release`** | **`eslint Release`** |
```

Comment:
I'm having a bit of trouble understanding this table. Is "Current Version" the equivalent of `eslint@latest`? I'm also not clear on what the "Releasing" column is meant to indicate. Is that the desired next version?

Reply:
> Is "Current Version" the equivalent of `eslint@latest`?

No, "Current Version" is as described on https://eslint.org/version-support/. For example, when we released `9.0.0-alpha.0`, `9.0.0-alpha.0` became the "Current Version".

> I'm also not clear on what the "Releasing" column is meant to indicate. Is that the desired next version?

Yes, that's the version that is going to be released.

Any ideas for better names to clarify this?

Reply:
Ah ok. "Current" on the version support page refers to the major release line, not a single version.

Looking this over, I think it's confusing to combine settings for `eslint` and `@eslint/js` in the same table. For ESLint, we cannot guarantee that 9.25.0 goes to 9.25.1 because it depends on the commits. I think it may be better to separate these into two tables, one for `eslint` and one for `@eslint/js`.

As for the wording, maybe this?

1. "HEAD Version"
2. "Desired Next Version"
3. `eslint` Release Option

Reply:
Updated in https://github.com/eslint/eslint/pull/18871/commits/4482729e2048d9da083c7a33d988a1d55a0b8209